test/test_server.py

```diff
@@ -502,6 +502,20 @@ def test_http(self):
 class TestTransparent(tservers.TransparentProxTest, CommonMixin, TcpMixin):
     ssl = False
 
+    def test_tcp_stream_modify(self):
+        self.master.load_script(
+            tutils.test_data.path("scripts/tcp_stream_modify.py"))
+
+        self._tcpproxy_on()
+        d = self.pathod('200:b"foo"')
+        self._tcpproxy_off()
+
+        self.master.masterq.join()
+
+        assert any("bar" in m for m in self.master.log)
```

Comment:
We should check if the response (`d`) contains bar as response, screw the log. :smile: